hpc-unibe-ch/hpc-unibe-ch.github.io
# Update python.md

I use Python3 (I hope most users do) and `pip install --user pandas` (directly after logging into Ubelix and loading the workspace module of my group, bern3dlpx-deps) gave me the error `-bash: pip: command not found`. Instead `pip3 install --user pandas` worked for me.

# Closes issue(s)  
=> none

Fixes #?  
=> pip changed to pip3 because pip did not work for Python 3 (the default)

# How to test this pull request?   
=> please verify that this issue was not specific for my account (see the comment above, that I always load a specific module). This can be done by simply repeating the few commands in the screenshot

# Screenshots   
see attached. You see that I try to run a python script; pandas is not found. Then installing with pip does not work, but with pip3 does. I test afterwards by running my script again and now we see it goes through the pandas import succesfully and finds the next error (not relevant here).  
<img width="1275" alt="Screenshot 2021-11-23 at 11 35 26" src="https://user-images.githubusercontent.com/1239545/143012918-b5782a28-8182-41c1-9339-e6d4de0ea34c.png">


# Changes include

# Checklist  
- [x] I have updated the README.md (if available and necessary)
